appveyor.yml

```diff
@@ -1,80 +1,14 @@
-          CMAKE_GENERATOR: "Visual Studio 15 2017 Win64"
-          PROJECT_DIR: features/workspace/cmake
-        # Package Development flow
-        - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
-          CMAKE_GENERATOR: "Visual Studio 14 2015 Win64"
-          PROJECT_DIR: features/package_development_flow
-        - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
-          CMAKE_GENERATOR: "Visual Studio 15 2017 Win64"
-          PROJECT_DIR: features/package_development_flow
-        # Serializing your data with Protobuf
-        - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
-          CMAKE_GENERATOR: "Visual Studio 14 2015 Win64"
-          PROJECT_DIR: libraries/protobuf/serialization
-        - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
-          CMAKE_GENERATOR: "Visual Studio 15 2017 Win64"
-          PROJECT_DIR: libraries/protobuf/serialization
-        # Using Facebook Folly with Conan
-        - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
-          CMAKE_GENERATOR: "Visual Studio 15 2017 Win64"
-          PROJECT_DIR: libraries/folly/basic
-        # Introduction to Dear ImGui
-        - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
-          CMAKE_GENERATOR: "Visual Studio 14 2015 Win64"
-          PROJECT_DIR: libraries/dear-imgui/basic
-        - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
-          CMAKE_GENERATOR: "Visual Studio 15 2017 Win64"
-          PROJECT_DIR: libraries/dear-imgui/basic
-        # Using Conan with find_package and exported targets
-        - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
-          CMAKE_GENERATOR: "Visual Studio 15 2017 Win64"
-          PROJECT_DIR: features/cmake/find_package/exported_targets_multiconfig
-        # Using the cmake_find_package_multi generator
-        - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
-          CMAKE_GENERATOR: "Visual Studio 15 2017 Win64"
-          PROJECT_DIR: features/cmake/find_package/find_cmake_multi_generator_targets
-        # Emscripten
-        - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
-          CMAKE_GENERATOR: "Visual Studio 15 2017 Win64"
-          PROJECT_DIR: features/emscripten
-        # Python requires: reusing a conanfile
-        - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
-          CMAKE_GENERATOR: "Visual Studio 15 2017 Win64"
-          PROJECT_DIR: features/python_requires/reuse_conanfile
-        # Multi config package (N configs -> 1 package)
-        - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
-          CMAKE_GENERATOR: "Visual Studio 15 2017 Win64"
-          PROJECT_DIR: features/multi_config
-        - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
-          PROJECT_DIR: features/lockfiles/ci
-        # Integrate a build-system in Conan
-        - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
-          CMAKE_GENERATOR: "Visual Studio 15 2017 Win64"
-          PROJECT_DIR: features/integrate_build_system
+    PYTHON: "C:\\Python27"
```

Comment:
For some reason, Python3 can't load Conan generator for WAF: https://ci.appveyor.com/project/ConanOrgCI/examples/builds/26208045?fullLog=true#L1910

Of course, it's related to the fact that I'm running Python -> Batch -> Python -> Conan

Reply:
What about using the value of `sys.executable` instead of just `python`?

Reply:
sounds much better

Reply:
done